src/components/BigPicture/screens/AboutScreen.tsx

```
@@ -0,0 +1,88 @@
+import { useReduceMotion } from '@/hooks/useReduceMotion';
+
+import { FocusCard } from '../components/FocusCard';
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)